src/main.rs

```diff
@@ -83,17 +83,21 @@ async fn create(
 		// 2. create mmr
 		let ephem_msk = translate::str_to_bytes(&env::var("EPHEM_MSK").unwrap())
 			.map_err(|e| (Status::InternalServerError, e.to_string()))?; // TODO: replace with an hkdf? https://github.com/ideal-lab5/murmur/issues/13
-		let (payload, store) =
-			murmur::create(username.into(), seed.into(), ephem_msk, schedule, round_pubkey_bytes)
-				.map_err(|e| (Status::InternalServerError, MurmurError(e).to_string()))?;
+		let create_data = murmur::create(
+			seed.into(),
+			ephem_msk,
+			schedule,
+			round_pubkey_bytes,
+		)
+		.map_err(|e| (Status::InternalServerError, MurmurError(e).to_string()))?;
 
 		// 3. add to storage
-		db.write(username.into(), store.clone())
+		db.write(username.into(), create_data.mmr_store.clone())
 			.await
 			.map_err(|e| (Status::InternalServerError, e.to_string()))?;
 
 		// 4. return the call data
-		Ok(CreateResponse { payload: payload.into() })
+		Ok(CreateResponse { create_data })
```

Comment:
With the change to murmur.js, the CreateResponse should also contain the username, something like:

```
#[derive(Serialize)]
pub(crate) struct CreateResponse {
	pub(crate) create_data: CreateData,
	pub(crate) username: Vec<u8>,
}
```
and then here:
```
Ok(CreateResponse { create_data, username: username.into() })
```